packages/pipeline/src/utils/get_ohlcv_trading_pairs.ts

```diff
@@ -66,27 +85,29 @@ export async function fetchOHLCVTradingPairsAsync(
     });
 
     // fetch all tokens that are traded on 0x
-    const rawTokenAddresses: Array<{ tokenaddress: string }> = await conn.query(
+    const rawEventTokenAddresses: Array<{ tokenaddress: string }> = await conn.query(
         `SELECT DISTINCT(maker_token_address) as tokenaddress FROM raw.exchange_fill_events UNION
       SELECT DISTINCT(taker_token_address) as tokenaddress FROM raw.exchange_fill_events`,
     );
-    const tokenAddresses = R.pluck('tokenaddress', rawTokenAddresses);
+    const eventTokenAddresses = R.pluck('tokenaddress', rawEventTokenAddresses);
 
     // join token addresses with CC symbols
-    const allTokenSymbols: string[] = tokenAddresses
+    const eventTokenSymbols: string[] = eventTokenAddresses
         .map(tokenAddress => erc20CoinsIndex.get(tokenAddress.toLowerCase()) || '')
         .filter(x => x);
```

Comment:
What's the point of this `filter`?

Reply:
this gets rid of the empty strings in the previous step (or any falsey values). I can change it to be more explicit though

Reply:
Let's be more explicit. I don't like making assumptions based on JavaScript "truthiness" rules because they can be pretty unintuitive sometimes.